pkg/lint/rules/template.go

```diff
@@ -117,7 +117,7 @@ func Templates(linter *support.Linter, values map[string]interface{}, namespace
 func validateTemplatesDir(templatesPath string) error {
 	if fi, err := os.Stat(templatesPath); err != nil {
 		return errors.New("directory not found")
-	} else if err == nil && !fi.IsDir() {
```

Comment:
how come you took the `err == nil` check out here?

Reply:
nvm I re-read the logic and removing the nil check there makes sense because you could any not nil errors earlier.

Reply:
it is a NOOP.  The function returning the error has been refactored out